src/bin/stratisd.rs

```diff
@@ -109,8 +133,13 @@ fn run() -> StratisResult<()> {
             }
         }
 
-        // Ask the engine to check its pools
-        engine.borrow_mut().check()
+        // Refresh list of dbus fds to poll for every time. This can change as
+        // D-Bus clients come and go.
+        fds.truncate(dbus_fd_start_idx);
```

Comment:
Just set up dbus fds here (rather than adding to uber-list of fds.

Reply:
We can't do this because it all needs to be in the uber-list.